source/docs/hardware-components/kit-and-hardware-guide/index.rst

```diff
@@ -17,10 +17,9 @@ Which Kit Should One Choose?
 
 Choice of a kit is a matter of many a debate in FTC forums, and each team has their favorites. If you are a rookie team and do not have any experience with any of the kits above, we would recommend starting with a kit from either REV Robotics or goBILDA. These kits provide a good selection of parts, reliability, and value for money. They are easily expandable and great part flexibility allows for customization. Keep in mind that compatibility between kits varies and will be impacted by the unit system used.
 
-- REV Robotics (metric) is slightly cheaper but generally has a bit higher learning curve than most other kits, as it is :term:`extrusion <Extrusion>` rather than :term:`channel <Channel>` based. :term:`Extrusion` takes more effort to work with (you need to cut aluminum extrusion to length); however, it allows for maximum design freedom and adjustability. REV is also available from the *FIRST*\ |reg| storefront, which might be convenient for teams doing their purchasing through official school procurement systems.
 - goBILDA (metric) is slightly more expensive, but is easier to get started with. It has an extensive parts catalog and can be adapted to other build systems like REV. However, its price point may deter some teams with lesser budgets.
+- REV Robotics (metric) is slightly cheaper but generally has a bit higher learning curve than most other kits, as it is :term:`extrusion <Extrusion>` rather than :term:`channel <Channel>` based. :term:`Extrusion` takes more effort to work with (you need to cut aluminum extrusion to length); however, it allows for maximum design freedom and adjustability. REV is also available from the *FIRST*\ |reg| storefront, which might be convenient for teams doing their purchasing through official school procurement systems.
```

Comment:
last sentence feels rambly. could cut a few weasel words here.

also is Robits worth a mention? It's also a storefront kit now.

Reply:
I think its fine? I can workshop it a bit but it gets the main points across.

I'm holding off on mentioning Robits because its so new, lets let the errata and other quirks stabilize before we talk too much about it.